postgres/gradle.properties

```diff
@@ -0,0 +1,4 @@
+ktor_version=2.2.3
```

Comment:
If you use the Ktor Gradle plugin, the dependencies of Ktor don't need to be declared with an explicit version.

Reply:
thank you, done